android/build.gradle

```diff
@@ -2,18 +2,18 @@
 
 buildscript {
     ext {
-        buildToolsVersion = "30.0.2"
+        buildToolsVersion = "33.0.0"
```

Comment:
Tools can be in a higher version as they are backwards compatible, so we can take the latest.